bigtable/client/row.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BIGTABLE_CLIENT_ROW_H_
+#define BIGTABLE_CLIENT_ROW_H_
+
+#include <absl/strings/string_view.h>
+#include <bigtable/client/cell.h>
+#include <vector>
+#include "bigtable/client/version.h"
```

Comment:
All Bigtable headers should go before system headers, to avoid implicit dependencies.

I think the ordering should be:

1. Bigtable headers
1. Generated headers (e.g., protobuf)
1. Third-party headers, such as Abseil
1. System/STL headers

Thoughts?

Reply:
Fundamentally I agree, and that is the proposal in #98.  However, that is not the order in the Google Style Guide, and clang-format is currently configured to follow that style guide.  When we adopt a new style guide we will change the clang-format configuration to enforce the right order.

Reply:
I think that is exactly what happened, I ran clang-format on it and version.h got pushed down. Fixed it by leaving an empty row.